public/app/features/dashboard-scene/edit-pane/DashboardEditableElement.tsx

```diff
@@ -21,6 +22,10 @@ export class DashboardEditableElement implements EditableDashboardElement {
     // When layout changes we need to update options list
     const { body } = dashboard.useState();
 
+    // Probably a nicer way to do this :(
+    const layoutOrchestrator = body as LayoutOrchestrator;
```

Comment:
I'm more fond of the pattern

```
if(!(body instanceof LayoutOrchestrator)){
   //handle the case where this is something other than LayoutOrchestrator
   //shouldn't happen, but if it does throw or return early
}
```

That way we don't run into weird bugs later downstream if body is something weird.

Reply:
Just didn't want a "rules of hooks" error due to using useState conditionally, but I guess it's a pattern we already use in scenes, so I'll go ahead and make the change

Reply:
Ah right we can't return early, so maybe just throw?